### DISCUSSION TITLE
Mention that Dev UI not available for new project not having RESTeasy dependency?

### DISCUSSION THREAD

**Discussion by ppatierno** [quarkusio/quarkus#22529] [Q&A]
I created a Quarkus application by using the VSCode extension and selecting the Kafka Streams dependency only.
I also select to "not include starter code".
It means no `GreetingResource` class as well as no dependencies like the `quarkus-resteasy` one.
At same time, the generated README.md mentions the following:

You can run your application in dev mode that enables live coding using:
```shell script
./mvnw compile quarkus:dev
```

> **_NOTE:_**  Quarkus now ships with a Dev UI, which is available in dev mode only at http://localhost:8080/q/dev/.

This is not always true.
AFAIU, it's because without the RESTeasy dependency, the Quarkus app cannot expose any HTTP endpoint and it makes sense.
I had to add just the `quarkus-resteasy` dependency to get the Dev UI working.
Did I get it right? Should it be mentioned in the generated README or maybe it's mentioned in some part of the doc but I missed it?

**Comment by maxandersen** [comment]
Its less than resteasy that is required. It needs a web endpoint so vertx should be enough.

**Reply by ppatierno** [reply]
So what should be the minimum dependency to be needed? Should be it mentioned somewhere in the doc? I would say it's useful. Maybe I just missed that?
